### PR TITLE
config: add core.checksum_jobs config option

### PR DESCRIPTION
**Pull request by Witiko**
Documents the `core.checksum_jobs` config option added by https://github.com/iterative/dvc/pull/2197.

### CODE REVIEW

File: static/docs/commands-reference/config.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ This is the main section with the general config options:
   [anonymized usage statistics](/doc/user-guide/analytics). Accepts values
   `true` (default) and `false`.
 
+- `core.checksum_jobs` - number of threads for computing checksums. Accepts
```
Comment (shcheklein) [inline static/docs/commands-reference/config.md]: @Witiko can we mention what is the default value for this, please? (even if it's a cpu()*something formula)

Comment (Witiko) [reply static/docs/commands-reference/config.md]: Would it be ok if we just say that it is “some sane value”? The default value is `None` and although `RemoteBASE._collect_dir` uses the `max(1, min(4, cpu_count() / 2))` formula, future parts of the code may use other sane default.

Comment (shcheklein) [reply static/docs/commands-reference/config.md]: Let's just mention this - `By default it's calculated by this (or similar) formula: ...? "`. It should give enough info - that it's something sane, conservative and related to cpu count.

**Comment by Witiko** [comment]
I updated the text as suggested.

**Comment by shcheklein** [comment]
@Witiko thanks! 🎉

**Pull request merged**